source/form.html

```diff
@@ -0,0 +1,148 @@
+              <label class="achievements-list__label" for="souvenirs">Накупил сувениров</label>
+            </li>
+            <li class="achievements-list__item">
+              <input class="achievements-list__input" id="tower" type="checkbox" name="achievements" value="tower">
+              <label class="achievements-list__label" for="tower">Удержал башню</label>
+            </li>
+            <li class="achievements-list__item">
+              <input class="achievements-list__input" id="hotel" type="checkbox" name="achievements" value="hotel">
+              <label class="achievements-list__label" for="hotel">Разгромил отель</label>
+            </li>
+          </ul>
+        </fieldset>
+
+        <fieldset class="competition__fields contact-data">
+          <legend class="visually-hidden">Контактные данные</legend>
+          <p class="contact-data__field">
+            <label class="contact-data__label" for="phone">Номер телефона</label>
+            <input class="contact-data__input" id="phone" type="tel" name="phone" placeholder="Номер, пожалуйста">
+          </p>
+          <p class="contact-data__field">
+            <label class="contact-data__label" for="email">Адрес почты</label>
+            <input class="contact-data__input" id="email" type="email" name="email" placeholder="Введите почту *">
+          </p>
+        </fieldset>
+
+        <fieldset class="competition__fields">
+          <legend class="competition__legend">С каким приложением путешествовали?</legend>
+          <ul class="competition__applications applications-list">
+            <li class="applications-list__item">
+              <input class="applications-list__input" id="iOS" type="radio" name="applications" value="iOS">
+              <label class="applications-list__label" for="iOS">Pink для iOS</label>
+            </li>
+            <li class="applications-list__item">
+              <input class="applications-list__input" id="android" type="radio" name="applications" value="android">
+              <label class="applications-list__label" for="android">Pink на Android</label>
+            </li>
+            <li class="applications-list__item">
+              <input class="applications-list__input" id="windows" type="radio" name="applications" value="windows">
+              <label class="applications-list__label" for="windows">Windows Phone</label>
+            </li>
+          </ul>
+        </fieldset>
+
+        <fieldset class="competition__fields emotions">
+          <legend class="emotions__legend">Опишите свои эмоции</legend>
+          <textarea name="emotions__textarea" placeholder="Можно прям в красках, не
+          стесняясь в выражениях"></textarea>
+        </fieldset>
+        <button class="competition__button" type="submit">Отправить форму</button>
+        <p class="competition__footnote">* — обязательные поля</p>
+      </form>
+    </main>
+
+    <footer class="page-footer page-footer--form">
```

Comment:
```suggestion
    <footer class="page-footer">
```

Футер не отличается от индекса же